src/content/docs/fr/2/guide/frontend/index.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Configuration Frontend
+
+i18nReady: true
+---
+
+import { LinkCard, CardGrid } from '@astrojs/starlight/components';
+
+Tauri est "frontend agnostique" et supporte la plupart des frameworks frontend prêts à l'emploi. Cependant, un framework a parfois besoin d'un peu plus de configuration pour l'intégrer à Tauri. Ci-dessous se trouve une liste des frameworks avec les configurations recommandées.

```

Comment:
```suggestion
Tauri est un "frontend agnostique" et supporte la plupart des frameworks frontend prêts à l'emploi. Cependant, un framework a parfois besoin d'un peu plus de configuration pour l'intégrer à Tauri. Ci-dessous se trouve une liste des frameworks avec les configurations recommandées.
```

Reply:
Are you sure about this one? It look weird to me to add `un`

Reply:
well i guess `"frontend agnostique"` is a noun phrase but it feels bad to me too, i'm gonna search for the french rule applying here

Reply:
ok seems like it depends on how we use it, if `frontend agnostique` is an expression, no problem to delete the `un`, if frontend is a common name and agnostique an adjective, we need `un` and maybe change the structure into `un "frontend" agnostique. Tbh I don't really know what it means, so is that a common expression in the programming sphere ?

Reply:
It's meaning is "frontend independent" where which frontend you choose doesn't really matter. Does that help clarify?

Reply:
> It's meaning is "frontend independent" where which frontend you choose doesn't really matter. Does that help clarify?

Yeah it actually do, it mean a common expresssion, and we can finally delete this `un`